crypto/o_time.c

```diff
@@ -47,6 +47,33 @@ typedef struct _ile3 {          /* Copied from ILEDEF.H for Alpha   */
 # endif   /* alpha || ia64    */
 #endif    /* OPENSSL_SYS_VMS  */
 
+time_t OPENSSL_timegm(struct tm *tm)
+{
+#if defined(__USE_MISC)
+    return timegm(tm);
+#else
```

Comment:
To avoid bit rot, we prefer this kind of construct:

``` C
    if (1) {
#if defined(__USE_MISC)
        return timemg(tm);
    } else {
#endif
    /* rest of code unconditionally compiled */
```

The idea is that a modern compiler is competent enough to optimize away the possibly dead code.